Bluetooth: Mesh: Added Ambient Light Sensor (#11127)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/sensor_client/README.rst

> The following Bluetooth mesh sensor types are used in this sample:

Reviewer: ```suggestion
The following Bluetooth mesh sensor types, and their settings, are used in this sample:
```

The Sensor Client model is now configured and able to receive data from the Sensor Server.

Interacting with the sample through shell
-----------------------------------------

1. Connect the development kit to the computer using a USB cable.
   The development kit is assigned a COM port (Windows), ttyACM device (Linux) or tty.usbmodem (MacOS).
#. |connect_terminal_specific_ANSI|
#. Enable local echo in the terminal to see the text you are typing.

> #. Enable mesh shell by typing "mesh init"

Reviewer: ```suggestion
#. Enable mesh shell by typing ``mesh init``.
```

> After completing the steps above, a command can be given to the client. See :ref:`bt_mesh_sensor_cli_readme` and :ref:`bluetooth_mesh_shell` for information about shell commands.

Reviewer: ```suggestion
After completing the steps above, a command can be given to the client.
See :ref:`bt_mesh_sensor_cli_readme` and :ref:`bluetooth_mesh_shell` for information about the shell commands.
```

SensorID/SettingID used in the shell commands are:

* :c:var:`bt_mesh_sensor_present_dev_op_temp` - 0x0054
* :c:var:`bt_mesh_sensor_dev_op_temp_range_spec` - 0x0013
* :c:var:`bt_mesh_sensor_rel_runtime_in_a_dev_op_temp_range` - 0x0064
* :c:var:`bt_mesh_sensor_presence_detected` - 0x004D
* :c:var:`bt_mesh_sensor_time_since_presence_detected` - 0x0069
* :c:var:`bt_mesh_sensor_motion_threshold` - 0x0043
* :c:var:`bt_mesh_sensor_present_amb_light_level` - 0x004E
* :c:var:`bt_mesh_sensor_gain` - 0x0074

> For example setting sensor gain for present ambient light level to 1.1 is:
> 
>    mesh models sensor setting-set 0x004E 0x0074 1.1

Reviewer: ```suggestion
For example, to set the sensor gain for present ambient light level to 1.1, write the following::

   mesh models sensor setting-set 0x004E 0x0074 1.1
```

The Sensor Client model is now configured and able to receive data from the Sensor Server.

Interacting with the sample through shell
-----------------------------------------

1. Connect the development kit to the computer using a USB cable.
   The development kit is assigned a COM port (Windows), ttyACM device (Linux) or tty.usbmodem (MacOS).
#. |connect_terminal_specific_ANSI|
#. Enable local echo in the terminal to see the text you are typing.
#. Enable mesh shell by typing ``mesh init``

After completing the steps above, a command can be given to the client.
See :ref:`bt_mesh_sensor_cli_readme` and :ref:`bluetooth_mesh_shell` for information about shell commands.

> SensorID/SettingID used in the shell commands are:

Reviewer 2: ```suggestion
Sensor Property ID or Sensor Setting Property ID that can be used in the shell commands are:
```

Reviewer 2: I think we should use the same terms as in the specification.

* :c:var:`bt_mesh_sensor_present_dev_op_temp` - 0x0054
* :c:var:`bt_mesh_sensor_dev_op_temp_range_spec` - 0x0013
* :c:var:`bt_mesh_sensor_rel_runtime_in_a_dev_op_temp_range` - 0x0064
* :c:var:`bt_mesh_sensor_presence_detected` - 0x004D
* :c:var:`bt_mesh_sensor_time_since_presence_detected` - 0x0069
* :c:var:`bt_mesh_sensor_motion_threshold` - 0x0043
* :c:var:`bt_mesh_sensor_present_amb_light_level` - 0x004E
* :c:var:`bt_mesh_sensor_gain` - 0x0074

> For example, to set the sensor gain for present ambient light level to 1.1, write the following:

Reviewer: ```suggestion
For example, to set the sensor gain for present ambient light level to 1.1, write the following::
```
